src/main/java/htsjdk/tribble/readers/TabixReader.java

```diff
@@ -430,7 +430,7 @@ public String next() throws IOException {
     }
 
     /**
-     * Return
+     * Get an iterator for an interval specified by the sequence id and begin and end coordinates
      * @param tid Sequence id
      * @param beg beginning of interval, genomic coords
      * @param end end of interval, genomic coords
```

Comment:
The javadoc for this method should document the convention being used for the genomic coordinates. Based on the callsites for this method, they appear to be BED-style (0-based, closed-open).

Reply:
Also applies to the other overloads of `query()`. It looks like some overloads use the 1-based closed convention, while others like this one use the 0-based closed-open convention.

Reply:
Also document the behavior with non-existent contigs, etc.